.travis.yml

```diff
@@ -132,17 +132,20 @@ script:
   - |
     if [[ -v HIVE_TESTS ]]; then
```

Comment:
Why not just add `set -e` before the `if`?

Reply:
It depends what do we want. Travis keeps running the remaining tests after failure if those are specified in separate branches. Also it is consistent with the other branches (we run 1 test suite per branch in product tests).